src/components/utils/PropertyChecker/PropertyChecker.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Checks if defined value is part of an array, if not that Element will be removed and an error is thrown.
+ * @param vueElement Vue Element: In this context it will be: this.$el
+ * @param possibleOptions Array that is checked if it is in the array
```

Comment:
possibleOptions:  An array of possible options.
actualValue: A value to check for inclusion in the possibleOptions array.